src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -527,3 +527,10 @@
     padding: 5px 0;
     margin-right: 10px;
 }
+
+/***** Media query print: Do not print the Toolbar. *****/
+@media print {
+    .sf-toolbar, .sf-error-toolbar {
```

Comment:
`.sf-error-toolbar` is useless here, as the error toolbar also has the `sf-toolbar` class